api/who/who.proto

```diff
@@ -57,6 +69,8 @@ message CaseStats {
   int64 recoveries = 5;
 
   string attribution = 6;
+
+  StatSnapshot[] timeseries = 8;
```

Comment:
confirming that this will respond with every single snapshot associated with the requested jurisdiction? no limit on the number of snapshots / no paging mechanism / no way to request specific days, correct?

Reply:
Correct.  Don't think we should make it complicated right now.